src/common/selftest/selftest_axis.h

```diff
@@ -11,12 +11,14 @@ typedef struct _selftest_axis_config_t {
     int8_t dir;
     float length;
     const float *fr_table;
+    float length_min;
```

Comment:
please reconsider member ordering to avoid alignment issues (biggest members first if possible)

Reply:
fixed